PHPUnit/Framework/TestResult.php

```diff
@@ -580,9 +580,13 @@ public function run(PHPUnit_Framework_Test $test)
         $errorHandlerSet = FALSE;
 
         if ($this->convertErrorsToExceptions) {
+            $errorLevels = E_ALL | E_STRICT;
+            if (PHP_VERSION_ID >= 70400) {
+                $errorLevels = E_ALL;
+            }
```

Comment:
I think it'd be safer to reverse the condition, and use E_STRICT only inside a condition block.

Reply:
@falkenhawk True, thank you!